test/sharness/t0125-twonode.sh

```diff
@@ -89,11 +89,6 @@ test_expect_success "set up tcp testbed" '
 	iptb init -n 2 -p 0 -f --bootstrap=none
 '
 
-# test multiplex muxer
-export LIBP2P_MUX_PREFS="/mplex/6.7.0"
-run_advanced_test "--enable-mplex-experiment"
-unset LIBP2P_MUX_PREFS
```

Comment:
So..., @diasdavid says this is the *only* multiplexer that works in both go and JavaScript so we shouldn't remove it.